### ISSUE TITLE
Import error seemingly related to pbr

### ISSUE THREAD

**Issue by LourensVeen** [jbarnoud/panedr#20]
Hi,

I'm installing panedr into a miniconda environment using pip, then running a script that uses it. So far that's gone fine, but with the new version it raises an exception on import:
```
  File "/home/lveen/.cerise/api/files/miniconda/lib/python2.7/site-packages/panedr/__init__.py", line 4, in <module>
    __version__ = pbr.version.VersionInfo('vermouth').release_string()
  File "/home/lveen/.cerise/api/files/miniconda/lib/python2.7/site-packages/pbr/version.py", line 457, in release_string
    return self.semantic_version().release_string()
  File "/home/lveen/.cerise/api/files/miniconda/lib/python2.7/site-packages/pbr/version.py", line 462, in semantic_version
    self._semantic = self._get_version_from_pkg_resources()
  File "/home/lveen/.cerise/api/files/miniconda/lib/python2.7/site-packages/pbr/version.py", line 449, in _get_version_from_pkg_resources
    result_string = packaging.get_version(self.package)
  File "/home/lveen/.cerise/api/files/miniconda/lib/python2.7/site-packages/pbr/packaging.py", line 848, in get_version
    name=package_name))
Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name vermouth was given, but was not able to be found.
```
As far as I can tell, there is indeed no `vermouth` in setup.cfg. Copy-paste error?

**Comment by jbarnoud** [comment]
Thank you. It was a copy-paste error indeed, but in `__init__.py`. It is fixed in version 0.5.2 that is deploying now.

**Comment by LourensVeen** [comment]
Thanks for the quick fix!
